server/cassconnection.js

```diff
@@ -0,0 +1,42 @@
+//tried to have some separation, but dealing with the async nature of this made it take longer than deadline would allow
+
+const cassandra = require('cassandra-driver');
```

Comment:
I know you might've changed this by now, but it seems that the server doesn't actually use this file--is that right? It just makes its own connection?